api/src/main/java/net/kyori/adventure/text/NBTComponentImpl.java

```diff
@@ -79,7 +79,7 @@ public int hashCode() {
     );
   }
 
-  static abstract class BuilderImpl<C extends NBTComponent<C, B>, B extends NBTComponentBuilder<C, B>> extends AbstractComponentBuilder<C, B> implements NBTComponentBuilder<C, B> {
+  abstract static class BuilderImpl<C extends NBTComponent<C, B>, B extends NBTComponentBuilder<C, B>> extends AbstractComponentBuilder<C, B> implements NBTComponentBuilder<C, B> {
```

Comment:
Is there a way to change the checkstyle configuration to keep this `static abstract`?

Reply:
Sure -- this is done by the `ModifierOrder` checker, which I can just take out of the config.

If you have a specific preferred order, let me know and I can toss that in to the extra checkers library.